python/feathub/processors/local/local_processor.py

```diff
@@ -20,9 +21,11 @@
     Any,
     Sequence,
 )
+from urllib.parse import urlparse
 
 import pandas as pd
 from dateutil.tz import tz
+from pyspark.sql import SparkSession
```

Comment:
Can we import Spark-related dependencies dynamically only if we need to use them?

Since we do not require users to always install feathub[spark] to use local processor, it will be useful to catch and report proper error (e.g. please install FeatHub with Spark to use xxx) if pyspark is not installed.